src/applications/disability-benefits/all-claims/tests/utils.unit.spec.jsx

```diff
@@ -715,4 +722,73 @@ describe('isAnswering781aQuestions', () => {
       expect(needsToAnswerUnemployability(formData)).to.be.false;
     });
   });
+
+  describe('concatIncidentLocationString', () => {
+    it('should concat full address', () => {
+      const locationString = concatIncidentLocationString({
+        city: 'Test',
+        state: 'TN',
+        country: 'USA',
+        additionalDetails: 'details',
+      });
+
+      expect(locationString).to.eql('Test, TN, USA, details');
+    });
+
+    it('should handle null and undefined values', () => {
+      const locationString = concatIncidentLocationString({
+        city: 'Test',
+        state: null,
+        additionalDetails: 'details',
+      });
+
+      expect(locationString).to.eql('Test, details');
+    });
+  });
+
+  describe('getFlatIncidentKeys', () => {
+    it('should return correct amount of incident keys', () => {
+      expect(getFlatIncidentKeys().length).to.eql(PTSD_INCIDENT_ITERATION * 2);
+    });
+  });
+
+  describe('getPtsdChangeFieldTitles', () => {
+    const uiSchema = {
+      field1: {
+        'ui:title': 'field1',
+      },
+      field2: {
+        'ui:title': 'field2',
+      },
+      field3: <div>test</div>,
```

Comment:
I think you want to nest this in a `ui:title` property like you've done for the other two field properties.